src/Media_Command.php

```diff
@@ -31,6 +31,9 @@ class Media_Command extends WP_CLI_Command {
 	 * [<attachment-id>...]
 	 * : One or more IDs of the attachments to regenerate.
 	 *
+	 * [--image-size=<image_size>]
```

Comment:
While it's not an explicit requirement, I think `--image-size=<image_size>` would be better as `--image_size=<image_size>`. _Most_ WP-CLI assoc args use the underscore (while flags use the dash)